resources/views/products/show.blade.php

```diff
@@ -0,0 +1,26 @@
+@extends('layouts.layout')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-xl-6">
+            <img src="{{ asset($product->gallery) }}" alt="This is product image." class="detail-img">
```

Comment:
When we wanna show an image in the public directory, we have to use asset().
asset() indicate all resource file in public directory automatically.